src/StatusQ/Components/StatusClickableTag.qml

```diff
@@ -0,0 +1,139 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype StatusAccountSelectorTag
+   \inherits Rectangle
+   \inqmlmodule StatusQ.Components
+   \since StatusQ.Components 0.1
```

Comment:
Shouldn't this actually be in `Controls`? Cause essentially it's a form control..